ui/components/app/signature-request-siwe/index.scss

```diff
@@ -0,0 +1,140 @@
+@import 'signature-request-header/index';
+@import 'signature-request-message/index';
+
+.signature-request {
+  display: flex;
+  flex: 1 1 auto;
+  flex-direction: column;
+  min-width: 0;
+
+  @media screen and (min-width: $break-large) {
+    flex: initial;
+  }
+}
```

Comment:
Looks like the only line of these styles that isn't being overwritten is `min-width: 0` you can probably remove them

<img width="1440" alt="Screen Shot 2022-04-26 at 6 07 24 PM" src="https://user-images.githubusercontent.com/8112138/165417998-2d6dda26-4e05-4900-a7db-fcbefe0e697e.png">

Reply:
changed this to `signature-request-siwe` to keep the components orthogonal